.gitignore

```diff
@@ -21,7 +21,6 @@ BenchmarkDotNet.Artifacts
 
 # gdUnit autogenerated
 addons/gdUnit4/GdUnitRunner.cfg
-gdunit4_testadapter/
```

Comment:
This is correct, the folder and test runner is generated by the test adapter when if missing.